pages/linux/latte-dock.md

```diff
@@ -0,0 +1,20 @@
+# latte-dock
+
+> Replacement dock for Plasma desktops, providing an elegant and intuitive experience for your tasks and plasmoids.
```

Comment:
```suggestion
> Replacement dock for Plasma desktop with an elegant and intuitive experience for your tasks and plasmoids.
```
Even though the main description mentions `desktops`, I think `desktop` is correct here and it is a typo there. This can be referred to as you read their README file.